frontend/config/webpack.config.dev.js

```diff
@@ -0,0 +1,275 @@
+const autoprefixer = require('autoprefixer');
```

Comment:
Please remove this file since `webpack.config.dev.js` was replaced by `webpackDevServer.config.js` in #102